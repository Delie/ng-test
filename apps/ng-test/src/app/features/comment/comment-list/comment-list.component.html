<app-standard-page>
  <h1>{{ 'Comments' | translate }}</h1>
  <p>
    This is just a little sandbox/test project I made to experiment with:
  </p>
</app-standard-page>
