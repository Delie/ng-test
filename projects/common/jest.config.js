const jestHelpers = require('./jest.helpers');
module.exports = jestHelpers.getJestConfig('@ng-test/common', false, '../../', 'libs/core');
