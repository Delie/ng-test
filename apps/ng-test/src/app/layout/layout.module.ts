import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { HeaderComponent } from './header/header.component';
import { FooterComponent } from './footer/footer.component';
import { StandardPageComponent } from './page/standard-page/standard-page.component';
import { TranslateModule } from '@ngx-translate/core';
import { RouterModule } from '@angular/router';

@NgModule({
  declarations: [HeaderComponent, FooterComponent, StandardPageComponent],
  imports: [CommonModule, TranslateModule, RouterModule],
  exports: [HeaderComponent, FooterComponent, StandardPageComponent]
})
export class LayoutModule {}
