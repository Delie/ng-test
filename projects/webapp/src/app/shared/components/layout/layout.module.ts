import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { HeaderComponent } from './header/header.component';
import { FooterComponent } from './footer/footer.component';
import { StandardPageComponent } from './page/standard-page/standard-page.component';
import { TranslateModule } from '@ngx-translate/core';
import { RouterModule } from '@angular/router';
import { FooterModule } from './footer/footer.module';
import { HeaderModule } from './header/header.module';
import { StandardPageModule } from './page/standard-page/standard-page.module';

@NgModule({
  imports: [CommonModule, TranslateModule, RouterModule, FooterModule, HeaderModule, StandardPageModule],
  exports: [FooterModule, HeaderModule, StandardPageModule]
})
export class LayoutModule {}
