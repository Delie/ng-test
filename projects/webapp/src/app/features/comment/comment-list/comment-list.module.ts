import { NgModule } from '@angular/core';
import { CommentListComponent } from './comment-list.component';
import { TranslateModule } from '@ngx-translate/core';
import { LayoutModule } from '../../../shared/components/layout/layout.module';

@NgModule({
  declarations: [CommentListComponent],
  imports: [TranslateModule, LayoutModule],
  exports: [CommentListComponent]
})
export class CommentListModule {}
