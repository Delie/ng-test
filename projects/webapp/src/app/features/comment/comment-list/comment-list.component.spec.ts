import { async, ComponentFixture, TestBed } from '@angular/core/testing';

import { CommentListComponent } from './comment-list.component';
import { SharedModule } from '../../../shared/shared.module';
import { LayoutModule } from '../../../layout/layout.module';
import { TranslateModule } from '@ngx-translate/core';
import { RouterModule } from '@angular/router';
import { APP_BASE_HREF } from '@angular/common';

describe('CommentListComponent', () => {
  let component: CommentListComponent;
  let fixture: ComponentFixture<CommentListComponent>;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      declarations: [CommentListComponent],
      imports: [SharedModule, LayoutModule, TranslateModule.forRoot(), RouterModule.forRoot([])],
      providers: [{ provide: APP_BASE_HREF, useValue: '/' }]
    }).compileComponents();
  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(CommentListComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  it('should create', () => {
    expect(component).toBeTruthy();
  });
});
