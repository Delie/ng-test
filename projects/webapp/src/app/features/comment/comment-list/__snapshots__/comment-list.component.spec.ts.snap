// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommentListComponent should create the component 1`] = `
<app-comment-list>
  <app-standard-page>
    <h1>
      
    </h1>
    <p>
       This is just a little sandbox/test project I made to experiment with: 
    </p>
  </app-standard-page>
</app-comment-list>
`;
